__tests__/jesttest.tsx

```diff
@@ -6,6 +6,10 @@ import * as LibraryEntryPoint from '../src/entry-point';
 import LibraryContainer from '../src/components/LibraryContainer';
 import LibraryItem from '../src/components/LibraryItem';
 
+
+// To run all tests with coverage report use command "npm run test"
+// To run jest tests without coverage report use command "npm run jest"
+
```

Comment:
These descriptions are no longer valid, and also they need to be moved under `ReadMe.md` if it isn't already there (we won't have test instructions inside a test file in general).